common/src/service/HttpClientHelper.cs

```diff
@@ -24,6 +25,7 @@ namespace Microsoft.Azure.Devices
 
     sealed class HttpClientHelper : IHttpClientHelper
     {
+        static readonly JsonSerializerSettings s_jsonSerializerSettings = new JsonSerializerSettings { DateParseHandling = DateParseHandling.DateTime, };
```

Comment:
The DateParseHandling property has a default value of DateTime. Could you please explain the advantage of setting it explicitly as above?

Reply:
@abhipsaMisra The problem is that, user can override default settings,  as described in https://github.com/Azure/azure-iot-sdk-csharp/issues/1122

Reply:
Are you setting the json serialization settings in your application? Can you share a code snippet that reproduces this issue.

Reply:
```csharp
JsonConvert.DefaultSettings = () =>new JsonSerializerSettings
            {
                DateParseHandling = DateParseHandling.DateTimeOffset,
            };
```